chat-components/src/components/notificationpane/NotificationPane.tsx

```diff
@@ -0,0 +1,135 @@
+                        {!props.hideNotificationIcon && (decodeComponentString(props.componentOverrides?.notificationIcon) || <Image
+                            className={notificationIconProps.className}
+                            id={notificationIconProps.id}
+                            src={notificationIconProps.src}
+                            alt={notificationIconProps.alt}
+                            tabIndex={-1}
+                            styles={notificationIconImageStyles}
+                        />)}
+                    </Stack>
+
+                    <Stack 
+                        styles={infoGroupStyles}
+                        dir={props.dir ?? "ltr"}>
+                        {!props.hideTitle && (decodeComponentString(props.componentOverrides?.title) || <Label
+                            className={props.titleClassName}
+                            styles={titleStyles}
+                            tabIndex={-1}
+                            id={elementId + "-title"}>
+                            {props.titleText}
+                        </Label>)}
+                        {!props.hideSubtitle && (decodeComponentString(props.componentOverrides?.subtitle) || <Label
+                            className={props.subtitleClassName}
+                            styles={subtitleStyles}
+                            tabIndex={-1}
+                            id={elementId + "-subtitle"}>
+                            {props.subtitleText}
+                        </Label>)}
+                        {!props.hideHyperlink && (decodeComponentString(props.componentOverrides?.hyperlink) || <Link
+                            className={props.hyperlinkClassName}
+                            ariaLabel={props.hyperlinkAriaLabel}
+                            id={elementId + "-hyperlink"}
+                            styles={hyperlinkStyles}
+                            hoverStyles={hyperlinkHoverStyles}
+                            target={"_blank"}
+                            rel={"noopener noreferrer"}
+                            href={props.hyperlinkHref}>
+                            {props.hyperlinkText}
+                        </Link>)}
+                    </Stack>
+
+                    {!props.hideDismissButton && (decodeComponentString(props.componentOverrides?.dismissButton) || <DismissButton
+                        {...dismissButtonProps}
+                        className={props.dismissButtonClassName}
+                        onClick={props.dismissButtonProps?.onClick}
+                        styles={dismissButtonStyles}
+                        hoverStyles={dismissButtonHoverStyles}
+                    />)}
+                </Stack>
+
+                <Stack horizontal
+                    styles={buttonGroupStyles}
+                    dir={props.dir ?? "ltr"}>
+                    {!props.hideCloseChatButton && <CloseChatButton
+                        {...closeChatButtonProps}
```

Comment:
I see some default props inside `CloseButton`. Can we create files like `defaultCloseButtonProps` and merge them here instead of checking them individually inside `CloseButton`? (`text={props.text ?? Texts.NotificationCloseChatText}`)

Reply:
This will help when creating links for documentation change later

Reply:
I've moved the default prop application to chat-widget (where the other default prop applications are as well). See `defaultChatDisconnectControlProps.ts` https://github.com/microsoft/omnichannel-chat-widget/pull/351/files#diff-baa95c5389a3f9b9ffd90dccce69c94a784a12d41fdd3b6cdbeffd68c1938708